### PR TITLE
cecd:u: stub GetCecStateAbbreviated

### PR DESCRIPTION
**Pull request by mailwl**
This PR allow to run **Dead or Alive Dimensions** (Full version)

### CODE REVIEW

**Comment by JamePeng** [comment]
http://3dbrew.org/wiki/CECD_Services

In my opinion!
First, Make the function name to match the 3dbrew page(API), "GetCecStateAbbreviated".
Second,Create a enum class CecStateAbbreviated is better than the dead code "1"

File: src/common/logging/log.h [hunk 1]
```diff
@@ -62,7 +62,7 @@ enum class Class : ClassType {
     Service_NIM,                ///< The NIM (Network interface manager) service
     Service_NWM,                ///< The NWM (Network wlan manager) service
     Service_CAM,                ///< The CAM (Camera) service
-    Service_CECD,               ///< The CECD service
+    Service_CECD,               ///< The CECD (SpeetPass) service
```
Comment (turbedi) [inline src/common/logging/log.h]: "StreetPass", not "SpeetPass"

**Comment by bunnei** [comment]
Can you squash this please?

**Comment by mailwl** [comment]
done

**Comment by bunnei** [comment]
LGTM

**Pull request merged**